config/install/docker.ci.acceptance.yml

```diff
@@ -25,6 +25,9 @@ sections:
         rest-request-validation-cache-warmup:
             command: 'vendor/bin/console rest-api:build-request-validation-cache'
 
+        controller-cache-warmup:
```

Comment:
looks strange
glue controllers are never used at acceptance tests

seems like you edited too much files
original PR did not introduce these changes, even having the same deploy files in place

Reply:
Suite non-split already has it in this file config/install/docker.ci.acceptance.yml:28. I was referring to suite non-split